Packs/cisco-meraki/Integrations/ciscomerakiv2/ciscomerakiv2.py

```diff
@@ -0,0 +1,515 @@
+    serial = args.get('serial')
+    updates = {
+        'name': args.get('name'),
+        'tags': argToList(args.get('tags')),
+        'lat': args.get('lat'),
+        'lng': args.get('lng'),
+        'address': args.get('address'),
+        'notes': args.get('notes'),
+        'moveMapMarker': argToBoolean(args.get('moveMapMarker')),
+        'switchProfileId': args.get('switchProfileId'),
+        'floorPlanId': args.get('floorPlanId')
+    }
+    res = client.update_device(serial=serial, updates=updates)
+    command_results = CommandResults(
+        outputs_prefix='Meraki.Devices',
+        outputs=res,
+        outputs_key_field='serial',
+        readable_output=tableToMarkdown(f'Updated device (Serial: {serial}):', res)
+    )
+    return_results(command_results)
+
+
+def meraki_claim_device_command(client: Client, args: dict):
+    network_id = args.get('networkId')
+    serials = args.get('serials')
+    serial_list = argToList(serials)
+    res = client.claim_device(network_id=network_id, serials=serial_list)
+    outputs = {
+        "networkId": network_id,
+        "serials": serials,
+        'success': False,
+        'errors': None
+    }
+    readable_output = f'Claim of Serial numbers {serials} from network ID {network_id} failed'
+    if res.status_code == 204:
+        outputs['success'] = True
+        readable_output = f'Claim of Serial numbers {serials} from network ID {network_id} succeeded'
+    elif res.status_code == 404:
+        errors = extract_errors(res)
+        outputs['errors'] = errors
+        readable_output = f'Claim of Serial number {serials} from network ID {network_id} failed:\n{",".join(errors)}'
+    command_results = CommandResults(
+        outputs_prefix='Meraki.ClaimDevices',
+        outputs=outputs,
+        outputs_key_field=['networkId', 'serials'],
+        readable_output=readable_output
+    )
+    return_results(command_results)
+
+
+def meraki_fetch_organization_uplinks_command(client: Client, args: dict):
+    organization_id = args.get('organizationId')
+    network_ids = argToList(args.get('networkIds'))
+    serials = argToList(args.get('serials'))
+    res = client.fetch_org_uplinks(organization_id=organization_id, network_ids=network_ids, serials=serials)
+    command_results = CommandResults(
+        outputs_prefix='Meraki.Uplinks',
+        outputs=res,
+        outputs_key_field='serial',
+        readable_output=tableToMarkdown(f'Uplinks (Organization ID: {organization_id}):', res)
+    )
+    return_results(command_results)
+
+
+def meraki_fetch_wireless_firewall_rules_command(client: Client, args: dict):
+    network_id = args.get('networkId')
+    number = args.get('number')
+    res = client.fetch_wireless_firewall_rules(network_id=network_id, number=number)
+    command_results = CommandResults(
+        outputs_prefix='Meraki.Firewall.Rules',
+        outputs=res,
+        outputs_key_field=['policy', 'protocol', 'destPort', 'destCidr'],
```

Comment:
the `destPort` defined in the yml as `Meraki.Firewall.Rules.dstPort`

**please fix in all 4 places**  that use the same syntax 
```suggestion
        outputs_key_field=['policy', 'protocol', 'dstPort', 'destCidr'],
```